csp-mixin/panels.libsonnet

```diff
@@ -899,5 +899,487 @@ local commonlib = import 'common-lib/common/main.libsonnet';
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
+            {
+              id: 'displayName',
+              value: 'Network throughput Received',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableReadBytes',
+          },
+          properties: [
+            {
+              id: 'unit',
+              value: 'decbytes',
+            },
+            {
+              id: 'custom.cellOptions',
+              value: {
+                type: 'color-text',
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
+            {
+              id: 'displayName',
+              value: 'Total Bytes count read',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableWriteBytes',
+          },
+          properties: [
+            {
+              id: 'unit',
+              value: 'decbytes',
+            },
+            {
+              id: 'custom.cellOptions',
+              value: {
+                type: 'color-text',
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
+            {
+              id: 'displayName',
+              value: 'Total Bytes count write',
+            },
+          ],
+        },
+      ])
+      + g.panel.table.queryOptions.withTransformations([
+        {
+          id: 'merge',
+          options: {},
+        },
+      ]),
+
+    gce_memory_utilization:
+      this.signals.gcpceOverview.memoryUtilization.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + this.signals.gcpceOverview.memoryUsed.asPanelMixin(),
+    gce_total_packets_sent_received:
+      this.signals.gcpceOverview.packetsSent.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.standardOptions.withUnit('short')
+      + this.signals.gcpceOverview.packetsReceived.asPanelMixin(),
+    gce_network_send_received:
+      this.signals.gcpceOverview.networkSent.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + this.signals.gcpceOverview.networkReceived.asPanelMixin(),
+    gce_bytes_read_write:
+      this.signals.gcpceOverview.diskBytesRead.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + this.signals.gcpceOverview.diskBytesWrite.asPanelMixin(),
+
+    gce_text_instances:
+      this.signals.gcpce.instancesText.as()
+      + commonlib.panels.generic.text.base.stylize(),
```

Comment:
Same, I couldn't find if there any asText type?

Reply:
there is no text panel in the commonlib, and no asText() in singals. I think you can just create a content inside text panel here directly, and you don't need a signal: 
```
    gce_text_instances:
       g.panel.text.new("Instances")
       + g.panel.text.options.withContent("Use this section to look at one instance at a time by picking a value in the *Instance* picker at the top of the dashboard.")
```
https://grafana.github.io/grafonnet/API/panel/text/index.html

Reply:
Thanks! I added it https://github.com/grafana/jsonnet-libs/pull/1336/commits/f1aacad278d399a8c0f6ec8ecc0430b81421d63d